pyproject.toml

```diff
@@ -21,6 +21,7 @@ pyyaml = "5.4"
 requests = "2.26.0"
 appdirs = "^1.4.4"
 curlify2 = "^1.0.1"
+gitlint = "^0.15.1"
```

Comment:
@hebertjulio oh, there is one more thing I've just noticed. Would you mind moving `gitlint` to the dev dependencies section. You can do it by running

```
poetry remove gitlint
poetry add -D gitlint
```

Also, I think we could set a specific version `gitlint = "0.15.1"` instead of `gitlint = "^0.15.1"`.

Thanks!!